packages/pages/src/common/src/template/internal/getTemplateFilepaths.ts

```diff
@@ -38,3 +49,89 @@ export const getTemplateFilepathsFromProjectStructure = (
       : [projectStructure.templatesRoot]
   );
 };
+
+const globalClientRenderFilename = "_client.tsx";
+const globalServerRenderFilename = "_server.tsx";
+
+/**
+ * Determines the client and server rendering templates to use. It first looks for a _client/server.tsx file in the scoped
```

Comment:
what is a "scoped path"?

Reply:
Scoped paths are for multi-brand sites that use subfolders. Slapshot added this.

Reply:
ah right. thanks